pkg/sql/exec_util.go

```diff
@@ -1748,6 +1748,8 @@ type BackupRestoreTestingKnobs struct {
 	// execution.
 	CaptureResolvedTableDescSpans func([]roachpb.Span)
 
+	RunBeforeBackupFlow func(ctx context.Context)
```

Comment:
nit: I notice that a lot of PR reviews for backup/restore are automatically assigned to SQL Queries due to editing this file. Is there a strong reason that backup/restor testing knobs need to live in this file, or `pkg/sql` in general?

Reply:
You're right, I don't immediately see a strong reason for this-- I'll move them in a follow up PR.